react/components/SearchBox.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+// @ts-ignore
+import { canUseDOM } from 'vtex.render-runtime'
+
+declare var global: {
+  __hostname__: string
+  __protocol__: string
+  __pathname__: string
+}
+
+const GoogleSearchBox = () => {
+  const hostname = canUseDOM ? window.location.hostname : global.__hostname__
+  const protocol = canUseDOM ? window.location.protocol : global.__protocol__
+  return (
+    <Helmet>
+      <script type="application/ld+json">
+        {`
+            "@context": "http://schema.org",
+            "@type": "WebSite",
+            "url": \`${protocol}//${hostname}/\`,
```

Comment:
Ops, add 
```suggestion
            "url": \`${protocol}//${hostname}/{__RUNTIME__.rootPath || ''}\`,
```

Make sure this doesnt break SSR